packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -88,15 +88,16 @@ export default class MainProcess {
       },
     });
```

Comment:
Could you add comments to both instances of `stdio: 'pipe'` that stdio must be set to pipe as we read the address of the gRPC server from stdout? That's valuable insight and I think it should get written down in the code.

Reply:
Done